app/requirements.txt

```diff
@@ -1,9 +1,12 @@
-asgiref==3.5.0
+asgiref==3.6.0
+certifi==2023.5.7
+charset-normalizer==3.1.0
 Django==4.2.1
-djangorestframework==3.13.1
-psycopg2==2.9.3
-pytz==2022.1
+djangorestframework==3.14.0
+idna==3.4
+psycopg2-binary==2.9.6
+pytz==2023.3
 PyYAML==6.0
-requests==2.28.0
+requests==2.30.0
 sqlparse==0.4.4
-uritemplate==4.1.1
```

Comment:
Hey @Aveline-art, thanks for working on this! Could you let me know what I should pay special attention to when reviewing this PR? For existing dependencies, I think I should double-check the version numbers to make sure they are correct, but for ones that you deleted or added, what should I know or do before approving the PR?

Reply:
I think for now, if you can confirm that it builds and runs with these dependencies on your device then we should be good to assume that it will work in a higher environment.

Thanks for checking in!

Reply:
Got it, thanks!